mne_connectivity/envelope.py

```diff
@@ -145,8 +155,33 @@ def envelope_correlation(data, combine='mean', orthogonalize="pairwise",
             if absolute:
                 corr = np.abs(corr)
             corr = (corr.T + corr) / 2.
+
         corrs.append(corr)
         del corr
 
+    # ravel from 2D connectivity into 1D array
+    corrs = [corr.flatten() for corr in corrs]
```

Comment:
It seems like this should happen after `fun(corrs)` for backward compat with `combine` as `callable`

Reply:
Wouldn't the "callable" still work on "axis=0"? Since we're just flattening the (N, N) connectivity arrays?

Reply:
We didn't put constraints on how the callable must work, so if for some reason it relied on operating on (N, N) shape then the order you have it in now will break it

Reply:
Okay that makes sense! Fixed the order